src/Countries/US/California/CaliforniaEmploymentTrainingTax/V20190101/CaliforniaEmploymentTrainingTax.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace Appleton\Taxes\Countries\US\California\CaliforniaEmploymentTrainingTax\V20190101;
+
+use Appleton\Taxes\Countries\US\California\CaliforniaEmploymentTrainingTax\CaliforniaEmploymentTrainingTax as BaseCaliforniaEmploymentTrainingTax;
+use Appleton\Taxes\Traits\HasWageBase;
+use Illuminate\Database\Eloquent\Collection;
+
+class CaliforniaEmploymentTrainingTax extends BaseCaliforniaEmploymentTrainingTax
+{
+    use HasWageBase;
+
+    const TAX_RATE = 0.001;
+    const WAGE_BASE = 7000;
+
+    public function getBaseEarnings()
+    {
+        if (($this->payroll->earnings + $this->payroll->ytd_earnings + $this->payroll->wtd_earnings) < self::WAGE_BASE) {
```

Comment:
i feel like this chunk of logic has been duplicated several times now, do we need to put it in one place?